src/client/src/orangehrmRecruitmentPlugin/pages/SaveInterviewPass.vue

```diff
@@ -0,0 +1,179 @@
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-card-container">
+      <oxd-text class="orangehrm-main-title" tag="h6">
+        {{ $t('recruitment.mark_interview_passed') }}
+      </oxd-text>
+      <oxd-divider />
+      <oxd-form :loading="isLoading" @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                :label="$t('recruitment.candidate')"
+                disabled
+                :value="interview.candidate"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                :label="$t('recruitment.vacancy')"
+                disabled
+                :value="interview.vacancy"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <oxd-input-field
+                :label="$t('recruitment.hiring_manager')"
+                disabled
+                :value="interview.hiringManager"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <oxd-input-field
+                :label="$t('recruitment.current_status')"
+                disabled
+                :value="interview.status"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :rows="2" :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item
+              class="orangehrm-save-candidate-page --span-column-2"
+            >
+              <oxd-input-field
+                v-model="interview.notes"
+                :label="$t('general.notes')"
+                type="textarea"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-divider />
+        <oxd-form-actions>
+          <oxd-button
+            display-type="ghost"
+            :label="$t('general.back')"
+            @click="onBack"
+          />
+          <submit-button :label="$t('recruitment.mark_interview_passed')" />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {APIService} from '@/core/util/services/api.service';
+import {navigate} from '@/core/util/helper/navigation';
+
+const interviewModel = {
+  id: null,
+  candidate: null,
+  vacancy: null,
+  hiringManager: null,
+  status: null,
+  notes: null,
+};
+
+export default {
+  name: 'ScheduleInterview',
+
+  props: {
+    candidateId: {
+      type: Number,
+      required: true,
+    },
+    interviewId: {
+      type: Number,
+      required: true,
+    },
+  },
+
+  setup(props) {
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      'api/v2/recruitment/candidates',
+    );
+    const http2 = new APIService(
```

Comment:
```suggestion
    const httpCandidate = new APIService(
      window.appGlobal.baseUrl,
      'api/v2/recruitment/candidates',
    );
    const http = new APIService(
```
just try to use a meaningful name